### ISSUE TITLE
Sound and Haptic feedback not working.

### ISSUE THREAD

**Issue by DongDian455** [FleksySDK/VirtualKeyboardSDK#80]
## Environment
Iphone: xr
IOS: 16.1.1

this is my code:
```
  override func createConfiguration() -> KeyboardConfiguration {
      .....
       ....
       feedback = FeedbackConfiguration(
                soundMode: FeedbackConfiguration.SoundMode
                    .sound(FeedbackConfiguration.ResourceSoundSet.defaultSoundSet),
                haptics: true
            )
            
           let config = KeyboardConfiguration(
            feedback: feedback,
            apps: appsConfig,
            license: license
        )   
        return configuration
    }
 
```
## Symptoms
when i switch to the background and stay for a while and return to the foreground, Sound and Haptic will become invalid. 
Need to restart the keyboard to restor.

**Comment by xevifar** [comment]
Hi @DongDian455 , could you share with us the SDK version that are you using ? Thx

**Comment by DongDian455** [comment]
The SDK version I am currently using is 4.18.2.

**Comment by xevifar** [comment]
hello @DongDian455 , we are able to reproduce this and we will work on solving this as soon as possible. 
I will update this task when it is done. Thank you for the reporting!

**Comment by xevifar** [comment]
Hello @DongDian455 , 
This is fixed on the version [4.19.1](https://docs.fleksy.com/sdk-ios/changelog/#v4191)
Please, let me know if this works for you. 

Thank you for the reporting.

**Comment by DongDian455** [comment]
The issue has been resolved in version 4.19.1
